test/parallel/test-repl-tab-complete.js

```diff
@@ -242,3 +244,16 @@ testMe.complete('cus', function(error, data) {
   completionCount++;
   assert.deepEqual(data, [['custom'], 'cus']);
 });
+
+// Make sure tab completion doesn't crash REPL with proxy objects
+putIn.run(['.clear']);
+
+putIn.run([
+  'var proxy = Proxy.create({});'
+]);
+
+testMe.complete('proxy.', function(error, data) {
```

Comment:
Please use either the `referenceErrors` or `completionCount` variable here (see the rest of the test). This test is known to fail silently.